pkgs/development/compilers/jetbrains-jdk/default.nix

```diff
@@ -1,16 +1,24 @@
-{ lib, openjdk11, fetchFromGitHub, jetbrains }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, jetbrains
+, openjdk17
+}:
 
-openjdk11.overrideAttrs (oldAttrs: rec {
+openjdk17.overrideAttrs (oldAttrs: rec {
   pname = "jetbrains-jdk";
-  version = "11_0_13-b1751.25";
+  version = "17.0.3-b469.37";
 
   src = fetchFromGitHub {
     owner = "JetBrains";
     repo = "JetBrainsRuntime";
     rev = "jb${version}";
-    sha256 = "sha256-TPNYZUkAoiZfp7Ci3fslKnRNGY1lnyIhXYUt6J31lwI=";
+    hash =
+     # Upstream issue: https://github.com/JetBrains/JetBrainsRuntime/issues/163
+     if stdenv.isDarwin then "sha256-ExRvjs53rIuhUx4oCgAqu1Av3CNAgmE1ZlN0srEh3XM="
```

Comment:
Do we know if this is deterministic? I can't find any reference to whether git handles case conflicts in a deterministic way or not.

Reply:
I don't have access to a macOS device, could you help with that?

Reply:
Also, the hash of the two files that clash is the same. (It doesn't matter which of the two copies git chooses, it will hash the same)